build.sbt

```diff
@@ -132,6 +154,9 @@ val `polynote-kernel` = project.settings(
     "net.sf.py4j" % "py4j" % "0.10.7",
     "org.scalamock" %% "scalamock" % "4.4.0" % "test"
   ),
+  distFiles := Seq(assembly.value) ++ (dependencyClasspath in Compile).value.collect {
+    case jar if jar.data.name.matches(".*scala-(library|reflect|compiler|collection-compat|xml).*") => jar.data
+  },
```

Comment:
Is this part basically for packaging scala library jars?

Reply:
Yeah, it's saying what things we need to copy to the distribution from that module.